app/src/main/java/com/ericg/pagingmovies/data/APIService.kt

```diff
@@ -1,10 +1,11 @@
 package com.ericg.pagingmovies.data
 
+import com.ericg.pagingmovies.util.Constants
 import retrofit2.http.GET
 import retrofit2.http.Query
 
 interface APIService {
-    @GET("movie/popular")
+    @GET(Constants.END_POINT)
     suspend fun getPopularMovies(
         @Query("page") page: Int = 0,
         @Query("api_key") api_key: String = "",
```

Comment:
`TIP`
Follow [this tutorial](https://guides.codepath.com/android/Storing-Secret-Keys-in-Android#hidden-in-buildconfigs) to learn how you can hide the API key instead of passing `""` when pushing to git.